src/main/js/component/general/audit/Details.js

```diff
@@ -54,7 +54,9 @@ class Details extends Component {
         if (this.state.values.errorStackTrace) {
             errorStackTrace = <TextArea inputClass="stackTraceContainer" label="Stack Trace" readOnly name="errorStackTrace" value={this.state.values.errorStackTrace}/>;
         }
-        const jsonPrettyPrintContent = JSON.stringify(JSON.parse(this.state.values.content), null, 2);
+        const defaultContent = Object.assign({}, {'warning': 'Content in an Unknown Format'});
+        const jsonContent = this.state.values.content || JSON.stringify(defaultContent);
```

Comment:
For the default content, we use JSON.stringify then we parse it and stringify it again? That seems sill. We should have an if to correctly display the real value if available, otherwise show the default